build/arm/linker/nrf52840/softdevice.ld

```diff
@@ -20,7 +20,7 @@ SOFTDEVICE_MINIMUM_RAM_BASE = 0x1628;
 SOFTDEVICE_MINIMUM_CODE_BASE = 0x26000;
 
 /* Reserving 16K in total */
-APP_RAM_BASE = 16K;
+APP_RAM_BASE = 32K;
```

Comment:
Some info on why such a value is required would be good to add here. This change also requires modifications to system-part1 linker.ld files (LENGTH is no longer correct) and possibly others.